lib/main_common.pm

```diff
@@ -2511,7 +2511,9 @@ sub load_ha_cluster_tests {
     set_var('USE_LVMLOCKD', 0) if (get_var('USE_LVMLOCKD') and is_sle('<15'));
 
     # Wait for barriers to be initialized except when testing HAWK as a client
-    loadtest 'ha/wait_barriers' unless (check_var('HAWKGUI_TEST_ROLE', 'client'));
+    # or Pacemaker CTS regression tests
+    loadtest 'ha/wait_barriers' unless (check_var('HAWKGUI_TEST_ROLE', 'client')) or
```

Comment:
the `or` shouldn't be inside the `unless` brackets? Like this: `unless (check_var('HAWKGUI_TEST_ROLE', 'client') or get_var('PACEMAKER_CTS_REG'));`?